pipeline/scripts/13-excelExport.R

```diff
@@ -398,39 +398,49 @@ ggsave(paste0(outdir, "/plots/IS_line_select_sum.png"), plot = IS_sum_selection_
 
 ### POSITIVE CONTROLS CHECK
 # these positive controls need to be in the samplesheet, in order to make the Pos_Contr.RData file
-positivecontrol_list <- c('P1002.1_Zscore', 'P1003.1_Zscore', 'P1005.1_Zscore')
+# Positive control samples all have the format P1002.x, P1003.x and P1005.x (where x is a number)
+
+# positivecontrol_list <- c('P1002.1_Zscore', 'P1003.1_Zscore', 'P1005.1_Zscore')
+column_list <- colnames(outlist)
+patterns <- c("^(P1002.)[[:digit:]]+_", "^(P1003.)[[:digit:]]+_", "^(P1005.)[[:digit:]]+_")
```

Comment:
`.` -> matches all characters, probably want to use `\.`

Reply:
You are correct. P1002x4 was also correct according to this regex. changed it to (double escape for R)
`patterns <- c("^(P1002\\.)[[:digit:]]+_", "^(P1003\\.)[[:digit:]]+_", "^(P1005\\.)[[:digit:]]+_")`